WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ConfigurableBundleProductView.swift

```diff
@@ -52,29 +53,30 @@ struct ConfigurableBundleProductView: View {
                     .shimmering()
                     .renderedIf(viewModel.bundleItemViewModels.isEmpty && viewModel.loadProductsErrorMessage == nil)
                 }
+                .padding(.horizontal, insets: safeAreaInsets)
                 .background(Color(.listForeground(modal: false)))
             }
-            .background(Color(.listBackground))
             .ignoresSafeArea(.container, edges: [.horizontal, .bottom])
```

Comment:
nit: I think we need to modify this line so it doesn't ignore the bottom safe area:

```suggestion
            .ignoresSafeArea(.container, edges: [.horizontal])
```

The `safeAreaInset` modifier is supposed to increase the safe area for the view it's modifying, to avoid overlap. Because `ScrollView` ignores the bottom safe area, the save button is overlapping the `ScrollView` content even when scrolled as far down as possible. I noticed this especially in landscape:

![Simulator Screenshot - iPhone 14 Pro - 2023-11-24 at 08 56 34](https://github.com/woocommerce/woocommerce-ios/assets/8658164/6201d49e-353c-4e7b-845f-397d13360388)

Increasing the safe area will still allow the button to overlap the content while scrolling, but when you scroll to the bottom of the `ScrollView` all the content will be above the button.

Reply:
That said, other lists like `ProductSelectorView` place the button directly below the scroll view (rather than using `safeAreaInset`) so the content never scrolls behind the button. Not sure if we want to use the same approach here for consistency.